dart/welcome_email/README.md

```diff
@@ -0,0 +1,37 @@
+# 📧 Sending Welcome Emails using Mailgun's Email API
+A sample Dart Cloud Function for sending a welcome email to a newly registered user.
+
```

Comment:
The readme definitely needs an update. Since this is outside the Appwrite context, we need to provide more information so terms like 'Cloud Functions' , `.appwrite` makes more sense.